ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -508,6 +508,34 @@ In AVD 4.0.0, we had "switched" as the default value for `ethernet_interfaces[].
 
 With AVD 5.0.0, the default value for `type` in `ethernet_interfaces` and `port_channel_interfaces` is no longer supported. The `type` key must now be explicitly defined in the input variables if it is needed in the configuration and documentation.
 
+### Removal of `type` key dependency for rendering ethernet/port-channel interfaces configuration and documentation
+
+#### IPv4 and IPv6 tables
+
+In AVD 4.0.0, the tables for `IPv4` and `IPv6` in ethernet and port-channel interfaces were rendered only if `ethernet_interfaces[].type` or `port_channel_interfaces[].type` was set to `routed/l3dot1q`.
+
+With AVD 5.0.0, the dependency on the 'type' key for rendering tables for IPv4 and IPv6 in ethernet and port-channel interfaces has been removed.
+
+#### `vlan_id` configuration
```

Comment:
Please add a `diff` example of how the old and new models look. Same for the others.

Reply:
Fixed